features/jdbc-collector/src/main/java/org/opennms/netmgt/config/jdbc/JdbcDataCollectionConfig.java

```diff
@@ -97,7 +97,28 @@ public JdbcDataCollection getDataCollectionByName(String name) {
         
         return null;
     }
-    
+
+    public JdbcDataCollectionConfig merge(JdbcDataCollectionConfig other) {
+        if (other == null) {
+            return this;
+        }
+        // Overwrite the rrdRepository iff it's null
+        if (m_rrdRepository == null && other.m_rrdRepository != null) {
+            m_rrdRepository = other.m_rrdRepository;
+        }
+        // Merge the lists of collections and groups
+        getCollection().addAll(other.getCollection());
+        //getGroup().addAll(other.getGroup());
```

Comment:
No groups for JDCB? If not, remove this line.

Reply:
Nope, no groups.  I've removed the commented line.